README.md

```diff
@@ -1,27 +1,163 @@
+- ask for reviews on community calls if this is a cross-sig VEP.
+
+#### SIGs
+
+The responsibility of the SIGs is to make sure the VEP is implemented, not diverging and that the implementation 
+is not lacking behind the VEP, following this non-exhaustive list meant as checklist:
+
+1. After Code Freeze, SIGs need to go over a tracking issue and perform the following checklist:
+   1. All PRs are merged into release branch
+   2. Docs PR is merged (plan review ahead of release if only placeholder is opened)
+   3. Verify that the Enhancement was implemented and doesn't need any update or exception
+   4. Track any bugs
+   5. Make sure the VEP's issue is tracking required PRs and Issues
+2. Weekly check-in on progress of the Enhancement and its implementation
+3. Coordinate SIGs, reviewers and approvers in order to progress the Enhancement
+
+### Release check-ins
+Both the release team and approvers of the VEPs are responsible for weekly check-ins, the outcomes of which will be posted on 
+the VEP's tracking issue. The following are the goals of the
+check-ins:
+
+1. Re-targeting of VEP - In case of implementation not converging, new blockers being discovered, pushback of community
+   or withdrawal of an approver the VEP needs to be re-targeted. The VEP needs to be updated with a new target. The SIGs
+   should shift the focus on tracked VEPs.
+   Re-targeting could also be rejection of the VEP completely in case it is not implementable.
+
+2. Coordination - SIGs are responsible to ensure reviews are not lagging behind by more than a week. 
+   The release team makes sure there is always an active SIG representative.
+
+### Labels
+
+For easier management of the release and VEPs the following labels will be used:
+1. SIG labels - Each SIG will have a label in order to sort which SIG is responsible for the VEP.
+2. Target labels - There will be a label in order to target the VEP for release.
+
+---
+**NOTE**
+
+Acceptance of an enhancement doesn't guarantee that the feature will land in the current or later release. The process is
+collaborative effort between contributors and approvers. Features not landing in the release branch prior to CF will need
+to file for an exception, see [Exceptions](#Exceptions)
+
+---
+
+## Deadlines
+
+The particular deadlines are always changing based on the release and are published here: [kubevrirt/sig-release](https://github.com/kubevirt/sig-release).  
+The following deadlines are important for the VEP:
+
+1. VEP planning - at the beginning of every release cycle, each SIG would prioritize VEPs and decide which ones are being tracked for the upcoming release.
+2. Enhancement Freeze - The deadline for this milestone is Alpha release of KubeVirt. See [kubevrirt/sig-release](https://github.com/kubevirt/sig-release/releases)
+3. Code Freeze - This is tracked by each release [kubevrirt/sig-release](https://github.com/kubevirt/sig-release/releases)
```

Comment:
Awesome! :tada: 
for the record: k8s have docs freeze and tests freeze, but I'm not sure we need those.

Reply:
For sure, we don't need them currently. :)